### DISCUSSION TITLE
Receive an Antivirus alert from Bitdefender

### DISCUSSION THREAD

**Discussion by OMacload** [Romanitho/Winget-AutoUpdate#520] [General]
Hi Romanito,

I setup Winget-Autoupdate since few months. But this week we received alert from Bitdefender regarding that : L'analyse à l'accès a détecté une menace. Le fichier a été supprimé. C:\ProgramData\
Winget-AutoUpdate
\WAU_update\WAU Configurator.lnk=>(CommandArguments) est un malware de type Heur.BZC.YAX.Boxter.800.3EA5A539

Do you have any information about that ?

Regards

Olivier

**Comment by Romanitho** [comment]
false positive. #519

**Reply by OMacload** [reply]
Thanks :)
